k8s/ietfweb/wagtail.yaml

```diff
@@ -0,0 +1,105 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: wagtail
+spec:
+  replicas: 1
+  revisionHistoryLimit: 2
+  selector:
+    matchLabels:
+      app: wagtail
+  template:
+    metadata:
+      labels:
+        app: wagtail
+    spec:
+      securityContext:
```

Comment:
must add `fsGroup: 1000` under `securityContext`. Otherwise you will have issues mounting the NFS volume.

Reply:
Addressed with https://github.com/ietf-tools/www/pull/453/commits/9038e7018b785db22de22719f212e27c62befcd7